### ISSUE TITLE
Issues trying to change the pool

### ISSUE THREAD

**Issue by Frax-Python** [UnamSanctam/UnamWebPanel#233]
im trying to change the pool on the web panel but when i change it nothing change, i already waited about 30 minutes

**Comment by UnamSanctam** [comment]
Then most likely the settings are wrong, post them here.

**Comment by Frax-Python** [comment]
pool.hashvault.pro:80

**Comment by Frax-Python** [comment]
Bro, the best monero pool is nicehash?

**Comment by UnamSanctam** [comment]
No, not by any means, NiceHash isn't technically a pool either, they are a hashrate marketplace. You just sell your hashrate to others when you mine to them. And if you're mining to hashvault then you'll have to use xmrig-proxy since if you have too many miners they will ban you (as it says on their homepage). But post the full configuration you're tying to use.

**Comment by Frax-Python** [comment]
{
    "algo": "rx/0",
    "pool": "pool.hashvault.pro",
    "port": 80,
    "wallet": "",
    "password": "",
    "nicehash": false,
    "ssltls": true,
    "max-cpu": `80,`
    "idle-wait": 2,
    "idle-cpu": 80,
    "stealth-targets": "Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe",
    "kill-targets": "",
    "stealth-fullscreen": false
}

**Comment by UnamSanctam** [comment]
Try another pool, you won't be able to use their pool anyways if you get more miners unless you don't set up your own xmrig-proxy server.
<img src="https://user-images.githubusercontent.com/10323655/212539127-7c793fbc-7e0c-4f20-9547-e7018c767fe5.png">

**Comment by Frax-Python** [comment]
ok, i think i will stay with nanopool
